pkg/graph/schema/types/translation.graphql

```diff
@@ -0,0 +1,62 @@
+"""
+Base typea that represents FE translation structure
+Translations are exported from FE for change history, and mapped to these types on the BE
+"""
+
+# Represents the data type of the translation field
+enum TranslationDataType {
+  STRING
+  NUMBER
+  BOOLEAN
+  DATE
+  ENUM
+  OBJECT
+}
+
+# Represents the FORM type of the translation field
+enum TranslationFormType {
+  TEXT
+  TEXTAREA
+  NUMBER
+  BOOLEAN
+  RADIO
+  CHECKBOX
+  SELECT
+  MULTISELECT
+  DATEPICKER
+  RANGEINPUT
+}
+
+# TranslationField represents a question with no options
+type TranslationField {
+  gqlField: String!
+  goField: String!
+  dbField: String!
+  label: String!
+  readonlyLabel: String
+  sublabel: String
+  multiSelectLabel: String
+  isArray: Boolean
+  dataType: TranslationDataType!
+  formType: TranslationFormType!
+  isOtherType: Boolean # Is a question a followup to another that doesn't designate it's own readonly question/line,
+  otherParentField: String # gql field name for the parent question for fields that represent Other, Please specify, etc.  Used in change history to render parent question for context
```

Comment:
Should this say the db_field name for the parent question? Or is that just dependent on the data export?

Also, if you use the ` ` ` comment style on the line before this field, it is exported in the introspection query

Reply:
The export converts it to the db field.  In the frontend, it is a gqlfield

Reply:
That works! I'm going to manually implement the type on the backend, so I can specify it's a db field name on the go type

Reply:
I'm also open to changing it to the db field, as currently the FE isn't using this in any way yet.  If it's just the BE use we can maybe convert that data

Reply:
Thanks Patrick! I'm good with trying any way to start it out. We could also just take out the GQL specific language, and just leave it to say field name. That way, it's true for FE and BE use cases?